### PR TITLE
add typescript method definitions to `.later`

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

File: lib/index.ts [hunk 1]
```diff
@@ -621,7 +636,7 @@ export default class Backburner {
     }
   }
 
-  private _setTimeout(target, method, args, wait) {
```
Comment (bekzod) [inline lib/index.ts]: also renamed `_setTimeout ` to `_later` to be consistent

**Comment by rwjblue** [comment]
CI is failing due to type issues:

```
lib/index.ts(404,12): error TS2556: Expected 1-6 arguments, but got a minimum of 0.
```

**Comment by rwjblue** [comment]
Maybe a rebase will fix this now?

**Comment by bekzod** [comment]
doing now

**Comment by bekzod** [comment]
now passes

**Pull request merged**